src/components/questions/RangeSlider/Notch.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { memo } from 'react';
+import { StyleSheet, View } from 'react-native';
+import COLORS from '../../../assets/colors';
+
+const Notch = ({ ...props }) => <View style={styles.root} {...props} />;
+
+export default memo(Notch);
```

Comment:
En hier ook memo, is dat specifiek ergens voor?

Reply:
Dit was hoe ze het in het voorbeeld van de package rn-range-slider deden